agentlocal/agentlocal.go

```diff
@@ -49,8 +49,10 @@ func SetTransport(ctx context.Context, debug bool) {
 type NetworkInfo bool
 
 const (
-	RequestNetworkInfo      NetworkInfo = true
-	DoNotRequestNetworkInfo NetworkInfo = false
+	RequestNetworkInfo        NetworkInfo = true
```

Comment:
Just wondering, why this linter error is resolved but not fixed? I see that the the `main` branch has no linter errors but this PR is about to bring a couple to it.

Reply:
@oksana-grishchenko Its weird, because this line were only formatted here, how it is possible that before it were ok? No other changes. And for example DoNotRequestNetworkInfo is same case. Thats why I resolved it.

Reply:
It's not about formatting, it's about the absent comment for a public constant. I would suggest either add comments or put `nolint`

Reply:
Yes I know, but in main it is without comment too. So for all public properties here should be comment.

Reply:
Oh, I see, that's really weird and not transparent behaviour. But anyway, I don't think it will disappear after merge to main. `nolint` will save the repo from extra errors

Reply:
Agree, added.